src/System.Xml.ReaderWriter/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -75,6 +68,7 @@ private static void AssertInvalidPath(string path)
             // Due to different shipping behavior in different products, different exceptions may be thrown
             // when an invalid path is specified on different platforms. We try to catch all the types
             // that can be thrown here to verify that more exception types are not introduced inadvertently.
+            // Ideally, expected exceptions should be parceled out by shipped product.
```

Comment:
Removed the comment (and updated the test to guarantee an error is thrown).

This was an attempt to say we should correlate which platforms throw which exceptions (to at least be aware of such changes), although I couldn't do that work myself.  Should I raise an issue?